src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -73,6 +75,7 @@ public PersonCard(Student student, int displayedIndex) {
         email.setText(student.getEmail().value);
         rate.setText(DOLLAR_SIGN + student.getRate().value + PER_HOUR);
         classTiming.setText(student.getClassTiming().value);
+        locationn.setText(student.getLocation().value);
```

Comment:
Maybe u can do this
```suggestion
location.setText("Location: " + student.getLocation().value);
```
So that people can tell the difference between the location and the student's address